OptionSelector/MainTableViewController.h

```diff
@@ -0,0 +1,15 @@
+//
+//  MainTableViewController.h
+//  OptionSelector
+//
+//  Created by Brian Blanco on 8/14/15.
+//  Copyright © 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface MainTableViewController : UITableViewController
+
+@property(strong, nonatomic) NSArray *animalCategories;
```

Comment:
Style: Object properties are `strong` by default, so you don't need to include `strong` in the property's attributes.